### ISSUE TITLE
Fix deprecateds

### ISSUE THREAD

**Issue by btd1337** [PlugaruT/wingpanel-indicator-sys-monitor#15]
Please fix this deprecateds

```
[1/12] Compiling Vala source ../src/Indicato...View.vala ../src/Widgets/DisplayWidget.vala.
../src/Widgets/SettingsView.vala:19.9-19.32: warning: Wingpanel.Widgets.Switch.switched has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
../src/Widgets/SettingsView.vala:21.37-21.62: warning: Wingpanel.Widgets.Switch.get_active has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
../src/Widgets/SettingsView.vala:24.9-24.32: warning: Wingpanel.Widgets.Switch.switched has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
../src/Widgets/SettingsView.vala:26.37-26.62: warning: Wingpanel.Widgets.Switch.get_active has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
../src/Widgets/SettingsView.vala:29.9-29.33: warning: Wingpanel.Widgets.Switch.switched has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
../src/Widgets/SettingsView.vala:31.38-31.64: warning: Wingpanel.Widgets.Switch.get_active has been deprecated since 2.0.5. Use Wingpanel.Widgets.Switch.active
```

**Comment by PlugaruT** [comment]
Thanks for reporting. Will be fixed one the June is released. But, you can try to fix it and a PR.

**Comment by PlugaruT** [comment]
Fixed in 560d4a8d6da73b8922ef91987381a315f9771f1f.
